### PR TITLE
Align LDAP data structure

### PR DESCRIPTION
**Pull request by jhsiaosuse**
- Change users information all to `ou=Users,dc=example,dc=org`
- Change groups information all to `ou=Groups,dc=example,dc=org`

Signed-off-by: JenTing Hsiao <jenting.hsiao@suse.com>

### CODE REVIEW

File: adoc/admin-security-ldap.adoc [hunk 1]
```diff
@@ -129,44 +129,44 @@ connectors:
     # server provides access for anonymous auth.
     # Please note that if the bind password contains a `$`, it has to be saved in an
     # environment variable which should be given as the value to `bindPW`.
-    # bindDN: uid=seviceaccount,cn=users,dc=example,dc=com
-    # bindPW: password
+    bindDN: CN=user-admin,OU=Users,DC=example,DC=org
+    bindPW: <k8s-admins-password>
```
Comment (innobead) [inline adoc/admin-security-ldap.adoc]: Suggest to change to <password of bind DN>

File: adoc/deployment-sysreqs.adoc [hunk 2]
```diff
@@ -123,6 +123,10 @@ link:https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/hardware
 |External
 |Dex (OIDC Connect)
 
+|32001
```
Comment (innobead) [inline adoc/deployment-sysreqs.adoc]: It seems incorrect formatting when preview by html mode. Please double confirm.

File: adoc/admin-security-ldap.adoc (hunk 1 above)
Comment (innobead) [inline adoc/admin-security-ldap.adoc]: Suggest to checnage <Password of Bind DN>

**Comment by innobead** [comment]
@c3y1huang @cclhsu please help review.

File: adoc/deployment-sysreqs.adoc (hunk 2 above)
Comment (jhsiaosuse) [reply adoc/deployment-sysreqs.adoc]: Thanks. Finally, I know what's `.N+` meaning.

**Comment by jhsiaosuse** [comment]
@innobead /request-to-rereview

**Comment by innobead** [comment]
@r0ckarong or @nkoranova please help review, thanks.

File: adoc/admin-security-ldap.adoc [hunk 3]
```diff
@@ -377,6 +377,11 @@ connectors:
       nameAttr: sAMAccountName
 ----
 
+base64 encoded PEM file can be generated by
```
Comment (nkoranova) [inline adoc/admin-security-ldap.adoc]:
```suggestion
base64 encoded PEM file can be generated by running:
```

File: adoc/deployment-bootstrap.adoc [hunk 4]
```diff
@@ -70,45 +70,45 @@ The IP/FQDN must be reachable by every node of the cluster and therefore 127.0.0
...
-Besides the LDAP connector you can also setup other connectors. For additional connectors, refer to the available connector configurations in the Dex repository: https://github.com/dexidp/dex/tree/master/Documentation/connectors.
+<3> LDAP server base64 encoded root CA certificate file (e.g. `cat <root-ca-pem-file> | base64 | awk '{print}' ORS='' && echo`)
+<4> Bind DN of user that can do user searches.
+<5> Password of the user.
+<6> Label of LDAP attribute users will enter to identify themselves (e.g. `username`).
+<7> BaseDN where users are located (e.g. `ou=Users,dc=example,dc=org`).
+<8> Filter to specify type of user objects (e.g. "(objectClass=person)").
+<9> Attribute users will enter to identify themselves (e.g. mail).
+<10> Attribute used to identify user within the system (e.g. DN).
+<11> Attribute containing email of users.
```
Comment (nkoranova) [inline adoc/deployment-bootstrap.adoc]:
```suggestion
<11> Attribute containing the user's email.
```

**Comment by jhsiaosuse** [comment]
@nkoranova updated! please help merge it, thx.

**Comment by innobead** [comment]
@nkoranova is it able to merge? Thanks.

**Comment by innobead** [comment]
@nkoranova @jordimassaguerpla don't forget this PR. We need to improve this document, I saw @nkoranova approved for a while, it should be ready to merge. Thanks.

**Pull request merged**